files/ja/web/html/element/meta/index.md

```diff
@@ -120,15 +120,15 @@ slug: Web/HTML/Element/meta
       - {{htmlattrxref("content", "meta")}} 属性に正の整数が 1 つだけ含まれている場合は、ページを再読み込みするまでの秒数。
       - {{htmlattrxref("content", "meta")}} 属性に正の整数と、その後に文字列 '`;url=`' と有効な URL がある場合は、別のページにリダイレクトするまでの秒数。
 
-      ##### アクセシビリティの考慮
-
-      `refresh` の値で設定されたページは、時間間隔が短すぎになるという危険性があります。画面の内側から読み上げるような支援技術を利用している人は、自動的にリダイレクトされる前にページの内容を読み、理解することができないかもしれません。また、弱視の人にとっても、突然に予告なくページ内容が更新されると、混乱させる可能性があります。
-
-      - [MDN Understanding WCAG, Guideline 2.1 explanations](/ja/docs/Web/Accessibility/Understanding_WCAG/Operable#guideline_2.2_—_enough_time_provide_users_enough_time_to_read_and_use_content)
-      - [MDN Understanding WCAG, Guideline 3.1 explanations](/ja/docs/Web/Accessibility/Understanding_WCAG/Understandable#guideline_3.2_—_predictable_make_web_pages_appear_and_operate_in_predictable_ways)
-      - [Understanding Success Criterion 2.2.1 | W3C Understanding WCAG 2.0](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-required-behaviors.html)
-      - [Understanding Success Criterion 2.2.4 | W3C Understanding WCAG 2.0](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-postponed.html)
-      - [Understanding Success Criterion 3.2.5 | W3C Understanding WCAG 2.0](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-no-extreme-changes-context.html)
+      > **注意：**
```

Comment:
Please don't translate `**Note:**` because Japanese translation doesn't have the magic word.
(and using `注意：` is sometimes incorrect)
```suggestion
      > **Note:**
```

Reply:
Good call, it looks like yari used to expect https://github.com/mdn/yari/blob/e500e2f18ed19973bdcb54d49f59deb5172ee5de/kumascript/macros/NoteStart.ejs#L20 but it sounds like that form is going away from the comments